target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -530,6 +530,10 @@ wndr3700)
 	ucidef_add_switch_port_attr "switch0" 5 led 2
 	;;
 
+anonabox-pro)
```

Comment:
You are right, this was verified by another person in IRC. Thank you. I will make the change.